main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,89 @@
+# Coding Permissions
+
+Write a json file with the permissions that the proposal will need. For example, [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) is the proposal Agoric created for the PSM contract:
+
+
+::: details Show example permissions file
+```
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "econCharterKit": true,
+    "provisionPoolStartResult": true,
+    "psmKit": true,
+    "chainTimerService": "timer"
+  },
+  "produce": {
+    "testFirstAnchorKit": true
+  },
+  "installation": {
+    "consume": {
+      "contractGovernor": true,
+      "psm": true,
+      "mintHolder": true
+    }
+  },
+  "instance": {
+    "consume": {
+      "economicCommittee": true
+    }
+  },
+  "issuer": {
+    "produce": {
+      "DAI": true
+    },
+    "consume": {
+      "DAI": true
+    }
+  },
+  "brand": {
+    "consume": {
+      "DAI": true,
+      "IST": true
+    },
+    "produce": {
+      "DAI": true
+    }
+  }
+}
+```
+:::
+
+## Consume Section
+
+In this section you need to set all the permissions that your contract will need to be able to use (i.e., "consume"). Some of the listed permissions in the example PSM permission file above are of general interest to most contracts, while others are more specific to the PSM contract.
+
+* **agoricNamesAdmin**: Grants write access to the Agoric name service. This permission is somewhat specific to the PSM contract.
+* **bankManager**: Grants access to bank-related functionality within ERTP and Zoe. Most contracts will need access.
```

Comment:
`bankManager` is closely related to ERTP (and to cosmos) but not to Zoe, so I suggest taking Zoe out.

And I think most contracts will not need this very powerful capability: It's the right to manipulate nearly all cosmos assets in the chain.

That is: it's the capability/right to make a "bank" for any address, where a "bank" is a per-address right to make an ERTP purse for any denom registered in the vbank... which raises the question of "what's the vbank?" so we might want to leave this level of detail out of scope.

Reply:
The description for bankManager has been changed to: 

"Grants access to bank-related functionality within ERTP, allowing the contract to manipulate nearly all Cosmos assets in the chain. Because this capability is very powerful, this permission should only be granted to contracts that absolutely need it."